zkevm-circuits/src/copy_circuit.rs

```diff
@@ -346,11 +353,12 @@ impl<F: Field> SubCircuitConfig<F> for CopyCircuitConfig<F> {
                     );
                 },
             );
-            cb.require_equal(
-                "write value == read value",
-                meta.query_advice(value, Rotation::cur()),
-                meta.query_advice(value, Rotation::next()),
-            );
+            // we use rlc to constraint the write == read now
```

Comment:
todo: constrain `rlc_acc_read` == `rlc_acc_write` when both src and dst memory tag.